overlord/ifacestate/helpers.go

```diff
@@ -98,13 +98,15 @@ func (m *InterfaceManager) addSnaps() error {
 	return nil
 }
 
-func (m *InterfaceManager) unbreakTheWorld() error {
-	// XXX: As a special measure to unbreak the world refresh seccomp security
-	// of all the snaps on the system. Later on this should be improved to
-	// either refresh all security backends or to know how to apply stashed
-	// (versioned) security that was preserved by snapd on upgrade to a new
-	// version of itself.
-
+// regenerateAllSecurityProfiles will regenerate the security profiles
+// for apparmor and seccomp. This is needed because:
+// - for seccomp we may have "terms" on disk that the current snap-confine
+//   does not understand (e.g. in a rollback scenario). a refresh ensures
+//   we have a profile that matches what snap-confine understand
+// - for apparmor the kernel 4.4.0-65.86 has an incompatible apparmor
+//   change that breaks existing profiles for installed snaps. With a
+//   refresh those get fixed.
```

Comment:
What was this change? I don't see any new rules to fix the profiles, so it isn't clear to me what this is doing...